spec/requests/api/v1/checkout_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CheckoutController, type: :request do
+    let(:user) { FactoryGirl.create(:user) }
+    let (:dish) { FactoryGirl.create(:dish) }
+    let(:credentials) { user.create_new_auth_token }
+    let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
+
+    describe 'GET /v1/checkout' do
+        it 'should display dishes in order that are not payed' do
+            @order = Order.create(user: user)
+            @order.add(dish, dish.price)
+
+            get '/api/v1/checkout', headers: headers
+            expect(response_json['order_items']).to eq [{ 'name' => 'Pizza', 'price' => 1.0, 'quantity' => 1, 'ready_time' => '10 Jan 16:32' }]
+            expect(response.status).to eq 200
+        end
+
+        it 'send error message if no dishes exist in order' do
+            @order = Order.create(user: user)
```

Comment:
This is repeated in the block above. Pls extract to a before block but, preferably, use "let" instead